samples/sample_existingproducts.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
I would just name it `samples/product.json` and for the new one `samples/product-new.json`